pkgs/by-name/ka/kakoune-lsp/Hardcode-perl.patch

```diff
@@ -0,0 +1,51 @@
+From 17cec4d94bc04bd02b91361ca72f561f1eebce60 Mon Sep 17 00:00:00 2001
+From: V <v@anomalous.eu>
+Date: Tue, 9 Apr 2024 06:15:00 +0200
+Subject: [PATCH] Hardcode perl
+
+---
+ rc/lsp.kak | 13 ++-----------
+ 1 file changed, 2 insertions(+), 11 deletions(-)
+
+diff --git a/rc/lsp.kak b/rc/lsp.kak
+index b173478..015c3ca 100644
+--- a/rc/lsp.kak
++++ b/rc/lsp.kak
+@@ -280,12 +280,8 @@ define-command -hidden lsp-menu -params 1.. %{
+ }
+ define-command -hidden lsp-menu-impl %{
+     evaluate-commands %sh{
+-        if ! command -v perl > /dev/null; then
+-            printf "fail %{'perl' must be installed to use the 'lsp-menu' command}"
+-            exit
+-        fi
+         echo >$kak_command_fifo "echo -to-file $kak_response_fifo -quoting kakoune -- %reg{a}"
+-        perl < $kak_response_fifo -we '
++        @perl@ < $kak_response_fifo -we '
```

Comment:
Preference: I'd prefer if this were `@perl@/bin/perl` so that the `substituteAll` would just be `inherit perl`.